core/src/main/scala/zio/config/ReadError.scala

```diff
@@ -106,9 +108,17 @@ sealed trait ReadError[A] extends Exception with NoStackTrace { self =>
         )
       )
 
+    def renderSourceError[A](err: ReadError.SourceError[A]): Sequential =
+      Sequential(
+        List(
+          Failure(s"SourceError: ${err.message}" :: Nil)
+        )
+      )
```

Comment:
Yea, this makes sense too. Coz source can be per field as well and can be part of accumulated read errors!